x-pack/plugins/apm/public/components/app/service_inventory/service_list/service_list.test.tsx

```diff
@@ -26,6 +30,17 @@ function Wrapper({ children }: { children?: ReactNode }) {
 describe('ServiceList', () => {
   beforeAll(() => {
     mockMoment();
+
+    const callApmApiSpy = getCallApmApiSpy().mockImplementation(
+      ({ endpoint }) => {
+        if (endpoint === 'GET /api/apm/fallback_to_transactions') {
+          return Promise.resolve({ fallbackToTransactions: false });
+        }
+        return Promise.reject(`Response for ${endpoint} is not defined`);
+      }
+    );
```

Comment:
afair: you can also use async/await and avoid the Promise wrappers. Not that it makes much of a difference

```suggestion
    const callApmApiSpy = getCallApmApiSpy().mockImplementation(
      async ({ endpoint }) => {
        if (endpoint === "GET /api/apm/fallback_to_transactions") {
          return { fallbackToTransactions: false };
        }
        throw new Error(`Response for ${endpoint} is not defined`);
      }
    );
```